src/vs/workbench/services/suggest/browser/media/suggest.css

```diff
@@ -294,7 +294,6 @@
 .workbench-suggest-widget .monaco-list .monaco-list-row > .contents > .main > .left {
 	flex-shrink: 1;
 	flex-grow: 1;
-	overflow: hidden;
```

Comment:
We still want overflow hidden as the left part's content could expand beyond the list item's container. Try tweaking the flex properties, we want left to take priority over the right side and I think they're equal currently.

Reply:
it actually never goes beyond the list item's container because there is a parent element with hidden overflow

Reply:
For example:

<img width="365" height="362" alt="Screenshot 2025-07-10 at 6 11 10 PM" src="https://github.com/user-attachments/assets/d336285b-f864-4220-8f65-32b833ccbbdb" />

Reply:
they're not equal since `left` will not hide on overflow and `right` will

Reply:
still seeing that one lack ellipses

<img width="361" height="208" alt="Image" src="https://github.com/user-attachments/assets/7127b6ee-aa7c-4792-b5ca-84ceb6b69644" />

Reply:
the way to get that to work is setting `flex-shrink` on the `.left` object. but doing so regardless of the other values then causes the left to shrink and the right takes over, which we don't want.

Reply:
figured it out